package/src/hooks/useStreami18n.ts

```diff
@@ -28,20 +31,21 @@ export const useStreami18n = (
       streami18n.addOnLanguageChangeListener((t) => {
         updateTFunction(t);
       });
+
     const { unsubscribe: unsubscribeOnTFuncOverrideListener } =
       streami18n.addOnTFunctionOverrideListener((t) => {
         updateTFunction(t);
       });
+
     streami18n.getTranslators().then((translator) => {
       if (translator && isMounted.current) setTranslators(translator);
     });
```

Comment:
this I think can be moved out of useEffect.. because there is no unsubscribe needed here..

So with useEffect this essentially runs after render rather than before render.. 

I hope it makes sense